dotcom-rendering/src/layouts/NewsletterSignupLayout.tsx

```diff
@@ -445,20 +445,22 @@ export const NewsletterSignupLayout = ({ article, NAV, format }: Props) => {
 										/>
 									</div>
 
-									<SecureSignup
-										name={promotedNewsletter.name}
-										newsletterId={
-											promotedNewsletter.identityName
-										}
-										successDescription={
-											promotedNewsletter.successDescription
-										}
-										hidePrivacyMessage={true}
-									/>
-
-									<Hide from="desktop">
-										<NewsletterPrivacyMessage />
-									</Hide>
+									<Island
+										priority="feature"
+										defer={{ until: 'visible' }}
+									>
+										<>
+											<SecureSignup
+												newsletterId={
+													promotedNewsletter.identityName
+												}
+												successDescription={
+													promotedNewsletter.successDescription
+												}
+											/>
+											<NewsletterPrivacyMessage />
+										</>
```

Comment:
I do not think you can have a child of island be anything but a single component that has `*.importable.tsx` in its filename